Library/Homebrew/dev-cmd/pull.rb

```diff
@@ -237,40 +243,72 @@ def pull
         safe_system "git", "commit", "--amend", "--signoff", "--allow-empty", "-q", "-m", message
       end
 
+      if testing_job
```

Comment:
`bottle_branch = if testing_job`

Reply:
Will change that; I do not have the ruby reflex to write code like that :) Thanks.